### DISCUSSION TITLE
Show sliderbars values above it instead of the bottom right

### DISCUSSION THREAD

**Discussion by Corne2Plum3** [ppy/osu#21334] [Ideas] [answered]
When using a sliderbar to change something in the mods customization menu or the options, the value is only displayed when the sliderbar is pressed and at the bottom right. The problem is, on mobile, we are like this:

![sliderbar1](https://user-images.githubusercontent.com/57914482/202913597-4c112bf6-a02a-478e-907f-53ca32200ef1.png)

As you can see, we can't see the value, or it's hard to see it, and it's weird. My suggestion would be to display the value **above** the sliderbar, so on mobile it would be easier to see.

![sliderbar2](https://user-images.githubusercontent.com/57914482/202913643-a91f50fa-a046-4db7-922e-0708ad70bff4.png)

Would also to think about it for all similar small textboxes, such as incompatibles mods or small informations such as the screenshot above ("Max combo").

![osu_2022-11-20_17-30-46](https://user-images.githubusercontent.com/57914482/202913831-d2dbeaf6-d2bd-4fb9-96f7-df1d833ce4eb.png)

**Comment by Joehuu** [comment, marked as answer]
Already tracked in https://github.com/ppy/osu/issues/4224.
